ansible/roles/test/tasks/crm/crm_test_ipv6_route.yml

```diff
@@ -13,6 +13,10 @@
     - name: Get NH IP
       shell: ip -6 neigh show dev {{crm_intf}} nud reachable nud stale | grep -v fe80
       register: out
+
+    - fail: msg="Get Next Hop IP failed. Neighbour not found"
+      when: out.stdout == ""
```

Comment:
I thinks this will never execute because grep will fail in previous task if didn't find anything.
Could you add an error message for this case?
Or better to ignore grep failure in previous task and fail here.

Reply:
Added `|| true` to follow grep.

Now if shell `ip -6 neigh show dev {{crm_intf}} nud reachable nud stale | grep -v fe80`,
if  failed will produce rc 0 and empty output, which will be handled by the following `fail:` operator.